spec/requests/users_spec.rb

```diff
@@ -0,0 +1,129 @@
+ require 'rails_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
+# It demonstrates how one might use RSpec to test the controller code that
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator. If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails. There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+
+RSpec.describe "/users", type: :request do
+  # User. As you add validations to User, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
+  }
+
+  let(:invalid_attributes) {
+    skip("Add a hash of attributes invalid for your model")
+  }
+
+  describe "GET /index" do
+    it "renders a successful response" do
+      User.create! valid_attributes
```

Comment:
factories?
also its just skipping the test when it hits https://github.com/mmmmaximus/rails-truck-logistics/pull/29/files#diff-7bd33678719b11d94cd425d48d7b45036f9f7ae51174c274abba505145f7fc3cR19